collector/introduction.md

```diff
@@ -8,6 +8,6 @@ When installed at the rig site, it has many advantages, such as high priority re
 
 ![](https://lh4.googleusercontent.com/MPecYS1S-WZ17EFA3fEhUms05m8FrdLzYm\_aRKAv0tMTSgML\_5N0xeIeRkePRDS29SLjgoDcE4b853K22V5F-Lq0EPhpL281ientTVuImNFakZJuGdH43LWie6MMGRmpEkNnZNVM)
 
-The collector primarily obtains data through the WITSML protocol; requesting log data from several service companies. Versions 1.3.1.1, 1.4.1.1 and 2.0 are supported. There is also support for the WITS protocol through TCP/IP or serial port, OPC-UA and OPC-DA.
+The collector primarily obtains data through the WITSML protocol; requesting log data from several service companies. Versions 1.3.1.1, 1.4.1.1 and 2.0 are supported. There is also support for the CSV and Raw protocol through TCP/IP or serial port, OPC-UA, OPC-DA and Modbus protocols.
```

Comment:
Why did we remove the WITS protocol from this phrase? WITS and WITSML are very different. Maybe we could change to something like:

```suggestion
The collector primarily obtains data through the WITSML protocol to request log, trajectory and other object data from service companies. WITSML versions 1.3.1.1, 1.4.1.1 and 2.0 are supported. 

Industry-standard protocols like OPC-UA, OPC-DA and Modbus are also supported. As a fallback for legacy system integrations, LiveRig Collector also can obtain data in WITS (level 0), CSV and Raw data through TCP/IP or serial ports.
```

Reply:
I was trying to rephrase it so that it doesn't look like WITS support serial port connection. But it looks like I accidentally erased the WITS protocol from the introduction. Anyway, the way you phrased it is way better, I will apply your suggestion. Thanks, Amadeu!

Reply:
Fixed in the commit 2ef49f35b175a80f91948b0480dabe96ad78bf21